js/main.js

```diff
@@ -18,9 +24,74 @@ function entryForm(event) {
   };
   data.nextEntryId++;
   data.entries.unshift(entry);
+  $ul.prepend(renderEntry(entry));
   $placeHolder.setAttribute('src', $placeHolderImage);
   $entryForm.reset();
+  viewSwap('entries');
+  toggleNoEntries();
+}
+
+function renderEntry(entry) {
+  const $entryItem = document.createElement('li');
+  $entryItem.setAttribute('class', 'row');
+
+  const $imageWrapper = document.createElement('div');
+  $imageWrapper.setAttribute('class', 'column-half');
+
+  const $image = document.createElement('img');
+  $image.setAttribute('src', entry.photoUrl);
+
+  const $contentWrapper = document.createElement('div');
+  $contentWrapper.setAttribute('div', 'column-half');
+
+  const $title = document.createElement('h3');
+  $title.textContent = entry.title;
+
+  const $notes = document.createElement('p');
+  $notes.textContent = entry.notes;
+
+  $entryItem.appendChild($imageWrapper);
+  $entryItem.appendChild($contentWrapper);
+  $imageWrapper.appendChild($image);
+  $contentWrapper.appendChild($title);
+  $contentWrapper.appendChild($notes);
+
+  return $entryItem;
+}
+
+function contentLoaded(event) {
+  for (let i = 0; i < data.entries.length; i++) {
+    const $entry = renderEntry(data.entries[i]);
+    $ul.appendChild($entry);
+  }
+}
```

Comment:
You should also be calling `viewSwap` inside of here passing in the last view the user was on, as well as calling `toggleNoEntries`